ui/ducks/bridge/bridge.ts

```diff
@@ -1,15 +1,22 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 import { swapsSlice } from '../swaps/swaps';
-import { RPCDefinition } from '../../../shared/constants/network';
+import { SwapsTokenObject } from '../../../shared/constants/swaps';
+import { SwapsEthToken } from '../../selectors';
+import { MultichainProviderConfig } from '../../../shared/constants/multichain/networks';
 
-// Only states that are not in swaps slice
 export type BridgeState = {
-  toChain: RPCDefinition | null;
+  toChain: MultichainProviderConfig | null;
+  fromToken: SwapsTokenObject | SwapsEthToken | undefined;
+  toToken: SwapsTokenObject | SwapsEthToken | undefined;
+  fromTokenInputValue: string | undefined;
 };
 
 const initialState: BridgeState = {
   toChain: null,
+  fromToken: undefined,
+  toToken: undefined,
+  fromTokenInputValue: undefined,
```

Comment:
Do we want to align on a common initial value between `null` and `undefined`?

`toChain` is `null` but the rest are `undefined`.